tox.ini

```diff
@@ -48,10 +48,14 @@ commands =
     py.test --create-db -n 2 -m 'not es_tests' -v src/olympia/ --ignore src/olympia/addons/ --ignore src/olympia/devhub/ --ignore src/olympia/editors/ --ignore src/olympia/amo/ --ignore src/olympia/users/ {posargs}
 
 [testenv:ui-tests]
-passenv = DISPLAY
-install_command = pip install {packages}
-deps = -rtests/ui/requirements.txt
-commands = pytest --verify-base-url --driver Firefox tests/ui {posargs}
+passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL ELASTICSEARCH_LOCATION
+install_command = pip install --exists-action=w --no-deps {packages}
+deps =
+    -rtests/ui/requirements.txt
+    -rrequirements/dev.txt
+commands =
+    make update_deps
```

Comment:
I suspect this needs to run before the install, which might be why @EnTeQuAk suggesting doing that within `commands`. I would suggest running this outside of Tox, perhaps in the Travis script.